src/components/sbb-checkbox/sbb-checkbox.tsx

```diff
@@ -173,10 +173,10 @@ export class SbbCheckbox extends LitElement {
   }
 
   public override willUpdate(changedProperties: PropertyValues<this>): void {
-    if (changedProperties.has('checked')) {
+    if (changedProperties.has('checked') && changedProperties.get('checked') !== undefined) {
```

Comment:
Do you use the additional check to skip the "willUpdate" that is called on initialization?
What if the consumer adds the `checked` attribute after the initialization?
I think that `!== undefined` might have unexpected effects

Reply:
Removed as I don't remember how to replicate the issue I had without it.